src/cascade/dismod/serialize.py

```diff
@@ -217,9 +218,22 @@ def observations_to_data(observations_df, node_table):
         eta=np.NaN,
         nu=np.NaN,
     )
-    keep = {"data_name", "integrand_id", "density_id", "node_id", "weight_id",
-            "hold_out", "meas_value", "meas_std", "eta", "nu", "age_lower",
-            "age_upper", "time_lower", "time_upper"}
+    keep = {
+        "data_name",
+        "integrand_id",
+        "density_id",
+        "node_id",
+        "weight_id",
+        "hold_out",
+        "meas_value",
+        "meas_std",
+        "eta",
+        "nu",
+        "age_lower",
+        "age_upper",
+        "time_lower",
+        "time_upper",
+    }
```

Comment:
I think this (and a few other style changes) are because I ran the pretty printer on this file. I've been trying to be careful to only do that in cases where it won't clutter up PRs or interfere with other folks' style choices but I seem to have run it here. That said I think I'm leaning towards long style as well. I used to argue against it but it looks ok to me now.